examples/foundry/image_classification.ipynb

```diff
@@ -0,0 +1,352 @@
+{
```

Comment:
<p>Could we add some brief text that explains what is the point of this step ? </p><p>Something like: This step is meant to generate annotations that can later be reused as pre-labels in a project</p><p>and link docs  perhaps ? https://docs.labelbox.com/docs/model-runs</p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Labelbox/labelbox-python/pull/1339/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>